.travis.yml

```diff
@@ -11,6 +11,8 @@ node_js:
   - "6.11"
   - "7.10"
   - "8.4"
+  - "8.8"
```

Comment:
We only keep one major; there is a duplicate 8.x in here. You can remove one of them.

Reply:
Well, the thing is that 8.8 added http2 without a flag, So the tests for 8.8 are different than for <= 8.7.

Reply:
Gotcha. So if you think both 8.x's should be kept, can a comment be added as to the reasoning so we don't remove it on accident? Also why keep 8.4 around, could that be 8.7 then? Here is an example of a version comment: https://github.com/mysqljs/mysql/blob/master/.travis.yml#L10

Reply:
Yes, should be 8.7 then. I'm not sure if both should be kept. But it might help at some point, possibly. Doesn't hurt, right?!

Reply:
> Doesn't hurt, right?!

haha, well, every version in there adds more time to wait for Travis CI to come back. That's the only thing it does, lol.